app/controllers/events_controller.rb

```diff
@@ -27,6 +27,18 @@ def index
     end
   end
 
+  # GET /events/calendar
+  # GET /events/calendar.js
+  def calendar
```

Comment:
Should this be a separate action, or should the `index` action take a parameter that determines which view is rendered?

Would avoid a lot of the JS stuff.

Reply:
We'll need the JS stuff anyway, since the calendar uses this to do move between months. I was thinking about this indeed, since we have the `:js` format free to use here, but it is a bit strange to expect this to render the calendar form.
Instead, what I think makes more sense, is to have another option on the calendar action to render the page directly. This would be nice as an iframe also.